descriptors/mongodb/debezium-mongodb-2.0.1.Final.json

```diff
@@ -162,6 +162,360 @@
+                            "type" : "string"
+                          },
+                          "cast_to" : {
+                            "title" : "Target type",
+                            "description" : "The target data type to cast to.",
+                            "$ref" : "#/$defs/kafka_connect_types"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }, {
+              "title" : "Cast payload fields",
+              "description" : "Cast specific field/s in a Kafka message key or value to another data type.",
+              "type" : "object",
+              "required" : [ "insert_static_field" ],
+              "additionalProperties" : false,
+              "properties" : {
+                "insert_static_field" : {
+                  "additionalProperties" : false,
+                  "required" : [ "apply_to", "field_name", "value" ],
+                  "properties" : {
+                    "apply_to" : {
+                      "$ref" : "#/$defs/apply_to"
+                    },
+                    "field_name" : {
+                      "$ref" : "#/$defs/insert_field_name"
+                    },
+                    "value" : {
+                      "type" : "string"
+                    }
+                  }
+                }
+              }
+            }, {
+              "title" : "Insert Kafka metadata field",
+              "description" : "Insert a field to the payload with Kafka record metadata as value (topic name, partition number, or message timestamp).",
+              "type" : "object",
+              "required" : [ "insert_kafka_field" ],
+              "additionalProperties" : false,
+              "properties" : {
+                "insert_kafka_field" : {
+                  "additionalProperties" : false,
+                  "required" : [ "apply_to", "kafka_value", "field_name" ],
+                  "properties" : {
+                    "apply_to" : {
+                      "$ref" : "#/$defs/apply_to"
+                    },
+                    "kafka_value" : {
+                      "title" : "Kafka record metadata type",
+                      "description" : "Choose which Kafka record metadata should be used as value for the inserted field (topic name, partition number, or message timestamp).",
+                      "type" : "string",
+                      "enum" : [ "topic name", "partition number", "message timestamp" ]
+                    },
+                    "field_name" : {
+                      "$ref" : "#/$defs/insert_field_name"
+                    }
+                  }
+                }
+              }
+            }, {
+              "title" : "Filter or rename fields",
+              "description" : "Filter out (excludes) or only pass (includes) specified fields in the payload, and/or rename field names.",
```

Comment:
@MelissaFlinn This is the description of the processor / SMT and what it is doing, not a description for a config option. I think this needs to be reworded in a way that is closer to the original text that describes the different actions that the user can achieve with this processor.

Reply:
I changed it to `"Filter out (excludes) or pass (includes) only the specified fields in the payload, and/or rename field names."` because I realized the sentence was a bit off....?!